<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCR识别标注 by HIT-DimWeaker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>标注区</string>
    </property>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>561</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>6</number>
     </property>
     <property name="midLineWidth">
      <number>4</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;图片&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>320</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="originEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>旧：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>新：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restoreButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>280</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>还原</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="preEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>测：</string>
     </property>
    </widget>
    <widget class="QLabel" name="preLabel">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>230</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>置信</string>
     </property>
    </widget>
    <widget class="QLabel" name="probLabel">
     <property name="geometry">
      <rect>
       <x>493</x>
       <y>250</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>120</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>标记区</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="uncertainBox">
        <property name="text">
         <string>不确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="invalidBox">
        <property name="text">
         <string>无效信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="punctuationBox">
        <property name="text">
         <string>有标点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="symbolBox">
        <property name="text">
         <string>有特殊符号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="spaceBox">
        <property name="text">
         <string>可能有空格</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="clarityBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>328</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 </string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>清晰度：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>切换区</string>
    </property>
    <widget class="QPushButton" name="lastButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>上一张</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>下一张</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lastUButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>上一张未标注</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextUButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>下一张未标注</string>
     </property>
    </widget>
    <widget class="QLabel" name="jumptextLabel">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>跳转编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="jumpButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>跳转</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>本地查看</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_local_save"/>
    <addaction name="action_remote_save"/>
    <addaction name="action_remote_load"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_local_save">
   <property name="text">
    <string>保存到本地</string>
   </property>
  </action>
  <action name="action_remote_save">
   <property name="text">
    <string>保存到服务器</string>
   </property>
  </action>
  <action name="action_remote_load">
   <property name="text">
    <string>从服务器读取</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
